spec/models/timer_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Timer, type: :model do
+  before(:each) do 
+    timer = Timer.new
+  end
+
+  let(:player) { FactoryGirl.create(:player, email: 'pam@rook.com', password: 'BESTKITTY') }
+  let(:game) { FactoryGirl.create(:game, game.id)}
+  let(:timer) { FactoryGirl.create(:timer, player_id: player.id)}
+
+    it "should determine when a player starts" do 
+      expect(DateTime.new).to be_a(DateTime)
+    end
+
+    it "should determine how much time the players have" do
+      @time_left = 500
+      expect(@time_left).to eq(500)
```

Comment:
This is a false positive. You're setting the `@time_left` to 500 right above this, there is no way for it to not be equal to 500

Also, you would want to be checking `timer.time_left`